web/src/app/profile/AlertComponent/AlertComponent.js

```diff
@@ -0,0 +1,58 @@
+import { Button, Form, FormGroup, FormControl } from 'react-bootstrap';
+import React, { Component } from 'react';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import PropTypes from 'prop-types';
+import styles from './AlertComponentStyles.scss';
+
+/* globals window */
+class AlertComponent extends Component {
+  render() {
+    const alertUrl = process.env.NODE_ENV === 'development' ?
+    'https://localhost:8080/?id=' + this.props.alertKey : process.env.PUBLIC_URL + '?id=' + this.props.alertKey;
```

Comment:
https://localhost:8080/ will need to be configurable. Lets move it to the .env file and make it a ENV_VARIABLE